packages/wallet/frontend/src/ui/Button.tsx

```diff
@@ -11,10 +11,17 @@ const buttonStyles = cva(
     variants: {
       intent: {
         primary: [
-          'border-transparent bg-green-dark dark:bg-pink-neon text-white'
+          'border-transparent border-none bg-green-dark dark:bg-pink-neon text-white'
```

Comment:
Why is the `border-none` needed?

Reply:
The size of the button changes when next to another type of button

Reply:
Let's revisit when we have the final designs as well.

Reply:
We need to consider figma designs final...we will not have any more input on this before the summit :(